sql/core/src/test/resources/sql-tests/inputs/columnresolution-views.sql

```diff
@@ -13,10 +13,8 @@ DROP VIEW view1;
 -- Test scenario with Global Temp view
 CREATE OR REPLACE GLOBAL TEMPORARY VIEW view1 as SELECT 1 as i1;
 SELECT * FROM global_temp.view1;
--- TODO: Support this scenario
 SELECT global_temp.view1.* FROM global_temp.view1;
```

Comment:
Yes, but @skambha did a comparison between different DB. Basically, what she tried to do is to make Spark SQL consistent with the others. See the design doc:
https://issues.apache.org/jira/secure/attachment/12854681/Design_ColResolution_JIRA19602.pdf